frontend/src/streamlist/__tests__/StreamsTable.test.jsx

```diff
@@ -150,7 +150,7 @@ describe("<StreamsTable/> component", () => {
         });
         it(`should have a render() function that returns a <StatusIndicator/> component`, () => {
           const dummyData = {
-            status: "Online"
+            streamInfo: dummyStreams[0]
           };
           const renderResult = result[4].render(dummyData);
```

Comment:
Why does this test need only the dummyData changed like this? Kinda sus ngl

Reply:
i think i changed the wrong dummyData because i went over this part too quickly LOL. dummyStream is needed for streamdetails
fixed 05cb998